disttask/framework/proto/task.go

```diff
@@ -96,13 +96,17 @@ type MinimalTask interface {
 const (
 	// TaskTypeExample is TaskType of Example.
 	TaskTypeExample = "Example"
+	// LoadData is TaskType of LoadData.
+	LoadData = "LoadData"
 )
 
 // Type2Int converts task type to int.
 func Type2Int(t string) int {
```

Comment:
it's more readable that we assign `TaskTypeExample` and `LoadData` a distinguish type, and associate these converting functions with the type